sleep/src/main.rs

```diff
@@ -11,7 +11,7 @@ fn main() {
         Some(values) => values,
         None => {
             eprintln!("sleep: Missing operand.\nTry 'sleep --help' for more information.");
-            process::exit(1);
+            process::exit(1);2
```

Comment:
Be careful with this, remember to check compiling for the whole workspace

Reply:
Okay... sorry about that. Must have commited that by accident.